homeassistant/util/color.py

```diff
@@ -1,12 +1,37 @@
 """Color util methods."""
+import logging
 import math
 # pylint: disable=unused-import
-from webcolors import html5_parse_legacy_color as color_name_to_rgb  # noqa
+
+_LOGGER = logging.getLogger(__name__)
 
 HASS_COLOR_MAX = 500  # mireds (inverted)
 HASS_COLOR_MIN = 154
 
 
+def color_name_to_rgb(color_name):
+    """Convert color name to RGB hex value."""
+    colors = {
```

Comment:
Please make this a constant in this module so that the Python interpreter will not be required to rebuild it every time this function is called.